etl/steps/data/garden/dummy/2020-01-01/dummy_full.meta.yml

```diff
@@ -9,6 +9,9 @@ tables:
         description_short: "[indicator.description-short]"
         description_from_producer: "[indicator.description-from-producer]"
         processing_level: "[indicator.processing-level]"
+        description_key:
+          - "[presentation.key-info-text example 1]"
+          - "[presentation.key-info-text example 2]"
         license:
```

Comment:
@pabloarosado from what I remember, the license should preferably be set at the origin level. Do we want to also include it at the indicator level?

Reply:
Yes, the license field is the license that we give to the indicator. We want to get rid of `licenses` at the VariableMeta class at some point because it is just the set union of the `Origin.license` of all indicators) but that depends on how much work it is.